modulus/utils/filesystem.py

```diff
@@ -47,6 +47,8 @@ def _get_fs(path):
 def _download_cached(path: str, recursive: bool = False) -> str:
     sha = hashlib.sha256(path.encode())
     filename = sha.hexdigest()
+
+    os.makedirs(LOCAL_CACHE, exist_ok=True)
```

Comment:
try-except block here to raise a warning in case this directory cannot created and suggesting to set the `LOCAL_CACHE` env var?

Reply:
Good call, added.